<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>collegeWindow</class>
 <widget class="QMainWindow" name="collegeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1066</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>College Window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="collegeTable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>1021</width>
      <height>401</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Code</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>College Name</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="prevButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>510</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Prev</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchBar">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>510</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pageNum">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>510</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1066</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="viewStudents"/>
    <addaction name="viewPrograms"/>
   </widget>
   <widget class="QMenu" name="menuManage">
    <property name="title">
     <string>Manage</string>
    </property>
    <addaction name="addCollege"/>
    <addaction name="deleteCollege"/>
    <addaction name="editCollege"/>
   </widget>
   <widget class="QMenu" name="menuSort">
    <property name="title">
     <string>Sort</string>
    </property>
    <addaction name="sortCode"/>
    <addaction name="sortName"/>
   </widget>
   <addaction name="menuView"/>
   <addaction name="menuManage"/>
   <addaction name="menuSort"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="viewStudents">
   <property name="text">
    <string>Students</string>
   </property>
  </action>
  <action name="viewPrograms">
   <property name="text">
    <string>Programs</string>
   </property>
  </action>
  <action name="addCollege">
   <property name="text">
    <string>Add College</string>
   </property>
  </action>
  <action name="deleteCollege">
   <property name="text">
    <string>Delete College</string>
   </property>
  </action>
  <action name="editCollege">
   <property name="text">
    <string>Edit College</string>
   </property>
  </action>
  <action name="sortCode">
   <property name="text">
    <string>Code</string>
   </property>
  </action>
  <action name="sortName">
   <property name="text">
    <string>Name</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
